Author: Daniel Baumann <daniel.baumann@progress-linux.org>
Description: Avoid overwriting build-environment.

diff -Naurp lzip.orig/configure lzip/configure
--- lzip.orig/configure
+++ lzip/configure
@@ -23,10 +23,10 @@ mandir='$(datarootdir)/man'
 build=no
 check=no
 installdir=
-CXX=g++
-CPPFLAGS=
-CXXFLAGS='-Wall -W -O2'
-LDFLAGS=
+#CXX=g++
+#CPPFLAGS=
+#CXXFLAGS='-Wall -W -O2'
+#LDFLAGS=
 
 # checking whether we are using GNU C++.
 /bin/sh -c "${CXX} --version" > /dev/null 2>&1 || { CXX=c++ ; CXXFLAGS=-O2 ; }
